xml/System.Security.Cryptography.Pkcs/Pkcs12Builder.xml

```diff
@@ -58,11 +58,26 @@
         <Parameter Name="pbeParameters" Type="System.Security.Cryptography.PbeParameters" />
       </Parameters>
       <Docs>
-        <param name="safeContents">To be added.</param>
-        <param name="passwordBytes">To be added.</param>
-        <param name="pbeParameters">To be added.</param>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <param name="safeContents">The contents to add to the PFX.</param>
+        <param name="passwordBytes">The byte array to use as a password when encrypting the contents.</param>
+        <param name="pbeParameters">The password-based encryption (PBE) parameters to use when encrypting the contents.</param>
+        <summary>Add contents to the PFX in an bundle encrypted with a byte-based password from a byte array.</summary>
+        <remarks>
+          <format type="text/markdown"><![CDATA[  
+  
+## Remarks  
+ The password bytes are passed directly into the Key Derivation Function (KDF) used by the algorithm indicated by `pbeParameters`.
+ This enables compatibility with other systems which use a text encoding other than UTF-8 when processing passwords with PBKDF2 (Password-Based Key Derivation Function 2).
+
+ The contents of `safeContents` are read during the call to this method, encrypted, and stored internally as encrypted content.
+ Any changes to the <xref:System.Security.Cryptography.Pkcs12.Pkcs12SafeContents> or any <xref:System.Security.Cryptography.Pkcs12.Pkcs12SafeBag> it references made after this method call are not reflected in the final output.
+
+ ]]></format>
+        </remarks>
+        <exception cref="T:System.ArgumentNullException">The <paramref name="safeContents"/> or <paramref name="pbeParameters"/> parameter is <see langword="null"/>.</exception>
+        <exception cref="T:System.ArgumentException">The <paramref name="safeContents"/> parameter value is already encrypted.</exception>
+        <exception cref="T:System.InvalidOperationException">The PFX is already sealed (<see cref="P:System.Security.Cryptography.Pkcs12.Pkcs12Builder.IsSealed"/> is <see langword="true"/>).</exception>
+        <exception cref="T:System.Security.Cryptography.CryptographicException"><paramref name="pbeParameters"/> indicates that <see cref="F:System.Security.Cryptography.PbeEncryptionAlgorithm.TripleDes3KeyPkcs12"/> should be used, which requires <see cref="T:System.Char"/>-based passwords.</exception>
```

Comment:
I'd be inclined to change "Char" to character (for language independence), except that linking to Char also seems to me to be a good idea. So feel free to ignore my comment if you want to leave the link to Char.